src/microbe_stage/editor/MicrobeEditor.cs

```diff
@@ -1183,9 +1183,6 @@ private void InitEditor()
         // Send info to the GUI about the organelle effectiveness in the current patch
         CalculateOrganelleEffectivenessInPatch(CurrentPatch);
 
-        // Reset this, GUI will tell us to enable it again
-        ShowHover = false;
```

Comment:
I don't think this is necessary as `ShowHover` defaults to false anyway and this resets the saved value. Should fix #1750.

Reply:
Wasn't this already in the code? I don't particularly think that this should be changed in an unrelated feature PR.

Reply:
Mean to say that this likely causes #1750 so I removed it, I'll separate the fix in another PR.

Reply:
Ah okay, yeah, having that as a separate PR would make reviewing things easier.